### PR TITLE
Added a Config for RefuelSpeed

### PR DESCRIPTION
**Pull request by ijzm**
Hello! I saw this simple mod the other day and really liked it. The only problem is that I thought the speed for refueling is way too slow.

I added a SimpleRefuel.cfg file, that allows any user to change the refuel speed. I set it as 10.0 as you had in your file before, but just changing that number will make things way faster.

I might look at being able to add more refueling zones accross Kerbin (for example in the Airfield, and the new launchpad)

### CODE REVIEW

**Comment by markusa380** [comment]
I'm happy you like the mod, and I think your work will be a great addition!
I will review it as soon as possible.

Can you give me your KSP forums account name, so I can add you to the list of contributors in the according thread?

**Comment by ijzm** [comment]
My account name is ijzm - https://forum.kerbalspaceprogram.com/index.php?/profile/187140-ijzm/

I'm glad you liked it. I might work a bit more on it later.

File: Source/Config.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SimpleRefuel {
+	public class Config {
+		private const string _configFilePath = "GameData/SimpleRefuel/SimpleRefuel.cfg";
```
Comment (markusa380) [inline Source/Config.cs]: Underscores should be used for temporary variables only. It is hardcoded anyway, so placing the literal string into the getter of ConfigFilePath makes more sense.

File: Source/Config.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		private const string SettingsNodeName = "SimpleRefuel_Settings";
```
Comment (markusa380) [inline Source/Config.cs]: This project is using camel case with small initial letters for variables.

File: Source/Config.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+		private static Config _instance;
```
Comment (markusa380) [inline Source/Config.cs]: Please do not use underscores in variables.

File: Source/Config.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+			get { return _instance ?? ( _instance = new Config() ); }
+		}
+
+		private Config() {
+			this._load();
+		}
+
+		internal float RefuelSpeed;
+
+		private void _load() {
```
Comment (markusa380) [inline Source/Config.cs]: Please do not use underscores for methods. Methods should have an uppercase first letter.

**Comment by markusa380** [comment]
I would be more than happy to see more pull requests from you! 😄 
Before I merge, can you change your variable naming according to my comments?
I am using the same naming convention as the KSP API does.
I would quickly do it myself, but I have no access to my Dev Environment at the moment.

**Review by markusa380 — CHANGES_REQUESTED** [review]

**Comment by ijzm** [comment]
I'm not going to be home for about 5 more hours,  but as soon as I get there I'll do those changes. Thanks for the feedback!

**Comment by ijzm** [comment]
Just did the changes you asked, hopefully everything is alright now. :D

**Pull request merged**